test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/protocolrestjson/AsyncOperationCancelTest.java

```diff
@@ -63,32 +63,35 @@ public void setUp() {
     }
 
     @Test
-    public void testNonStreamingOperation() {
+    public void testNonStreamingOperation() throws InterruptedException {
         CompletableFuture<AllTypesResponse> responseFuture = client.allTypes(r -> {});
         responseFuture.cancel(true);
+        Thread.sleep(500);
```

Comment:
Is 500 sufficient or will this cause transient failures?

Reply:
Not sure, I'm just trying to figure out if this will fix the test fails in Travis ATM. I'll probably increase to 1s if it works